staging/src/k8s.io/client-go/tools/cache/store.go

```diff
@@ -29,7 +29,7 @@ import (
 // queue of items yet to be processed.
 //
 // Store makes no assumptions about stored object identity; it is the responsibility
-// of a Store implementation to provide a mechanism to correctly key objects and to
+// of a Store implementation to provide a mechanism to correct key objects and to
```

Comment:
Can you elaborate on why this should be correct instead of correctly? To me this looks like it should remain as correctly.

Reply:
i think it should be a verb here，not a adj

Reply:
`key` is the verb here

Reply:
"implementation provide a mechanism to correctly define a key for objects and ..." which replaced "define a key for" with "key" which is ok to me. Closing this unless you think otherwise.